code-challenges.js

```diff
@@ -31,38 +67,92 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+const yourDegrees = (temp) => {
+    if (temp < 212) {
+        return `${temp} is below boiling point`
+    } else if (temp === 212) {
+        return `${temp} is at boiling point`
+    }
+    else if (temp > 212) {
+        return `${temp} is above boiling point`
+    }
+}
+console.log(yourDegrees(temp1))
+console.log(yourDegrees(temp2))
+console.log(yourDegrees(temp3))
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+// Dodgers are better...
+// Divide this into two parts: 1. combine the arrays 2.return the length
+// To combine the two arrays and preserve both of their values, use the ".concat" accessor. 
+    //Create a new variable "concatArrays" to describe the data being combined of the two variables provded.
+    // Set the new variable to padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+// Next, to determine the length of the new variable, use the ".push" mutator at the end of the new variable. 
+    // This will return 9 becuase nine numbers exist in the new array, since they were combined.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+var concatArrays =padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+// console.log(concatArrays)
+console.log(concatArrays.push())
```

Comment:
Interesting approach!  The length built in method could also come in handy as well.